mlflow/models/evaluation/base.py

```diff
@@ -1695,29 +1713,14 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
                     "parameter when model=None.",
                     error_code=INVALID_PARAMETER_VALUE,
                 )
-            if predictions not in data.columns:
-                raise MlflowException(
-                    message=f"The specified predictions column '{predictions}' is not "
-                    "found in the specified data.",
-                    error_code=INVALID_PARAMETER_VALUE,
-                )
```

Comment:
Removed since it's duplicate with
https://github.com/mlflow/mlflow/blob/2158f3ce96ab4e0e41de2e833cae1c962a14c684/mlflow/data/pandas_dataset.py#L54-L59